test/performance/README.md

```diff
@@ -1,128 +1,189 @@
+   - 10 secrets containing a 2048-character random string.
+   - 10 config maps
+
+   b) If using OpenShift clusters: 
+
+   ```bash
+   $ ./setup-clusterloader2.sh --env ocp
+   ```
+
+    In Openshift environments, a modified version of the kube-burner [cluster-density-ms](https://github.com/kube-burner/kube-burner-ocp/tree/478bb42e1842a94ca3210d26a08633b70a443005/cmd/config/cluster-density-ms) workload will be generated per namespace. This workload consists of the following objects:
+
+    - 1 image stream.
+    - 4 deployments, each with two pod replicas (pause), mounting 4 secrets, 4 config maps, and
+      1 downward API volume each.
+    - 2 services, each pointing to the TCP/8080 and TCP/8443 ports of the first and second
+      deployments, respectively.
+    - 1 edge route pointing to the first service.
+    - 20 secrets containing a 2048-character random string.
+    - 10 config maps containing a 2048-character random string
+
+
+3. Configure the parameters of your workload:  
 
    a) cd into the load configuration directory
 
-```bash
-$ cd  perf-tests/clusterloader2/testing/load
-```
-   b) create the `stress-test` namespace - you workload needs to be deployed into this exact namespace (temporary solution)
-
-```bash
-$ kubectl apply -f - <<EOF
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: stress-test
-  labels:
-    app.kubernetes.io/cluster: wec1
-EOF
-```
-
-   c) configure your sample workload: 
-   
-```bash
-$ vi test-workload.yaml
-```
+   ```bash
+   $ cd  $WORK_DIR
+   ```
   
-   d) configure the parameters to create the workload traffic (e.g., RandomizedLoad, SteppedLoad, Sequence, etc.)
+   b) configure the parameters to create the workload traffic (e.g., RandomizedLoad, SteppedLoad, Sequence, etc.)
    
-```bash
-$ vi performance-test-config.yaml
-``` 
+   ```bash
+   $ vi performance-test-config.yaml
+   ``` 
+
+   More specifically, configure the following parameters: 
+
+   - namespaces: number of namespaces to be created in your cluster to deploy the worklod defined in step-2 above (*default value: 1*)
+   - namespaceNamePrefix: name prefix for the created namespaces (*default value: "perf-exp"*). For example: if `namespaces=2` & `namespaceNamePrefix="perf-exp"`, then the following namespaces will be created: `perf-exp-0` and `perf-exp-1`
+   - K8S_CLUSTER: set to `true` for Kubernetes clusters (*default value: "true"*)
+   - OPENSHIFT_CLUSTER: set to `true` for Kubernetes clusters (*default value: "false"*)
+   - tuningSet: workload generation function (*default value: "RandomizedLoad"*)
+
+   To learn more about clusterloader2 workload generation functions, see the following: [clusterloader tuningSets](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/docs/design.md#tuning-set) and [configurations](https://github.com/kubernetes/perf-tests/blob/fac2a5eec96fab76a4bc4858795df4544b729b0b/clusterloader2/api/types.go#L249).
```

Comment:
"To learn more about clusterloader2 workload generation functions" does not go with a pointer to information about TuningSet. There is much more to clusterloader2 workload generation than TuningSet.